Sources/Application/Game/Game.swift

```diff
@@ -4,33 +4,38 @@ public class Game {
 	let view: Viewable
 	let maxNumberOfGuesses: Int
 	let guessBuilder: GuessBuilder
+	let players: [Player]
 	public var gameOver: Bool
 	public var winner: Bool
 
-	public init(word: String, guessManager: GuessManager, view: Viewable, maxNumberOfGuesses: Int, guessBuilder: GuessBuilder) {
+	public init(word: String, guessManager: GuessManager, view: Viewable, maxNumberOfGuesses: Int, guessBuilder: GuessBuilder, players: [Player]) {
 		self.word = word
 		self.guessManager = guessManager
 		self.view = view
 		self.maxNumberOfGuesses = maxNumberOfGuesses
 		self.guessBuilder = guessBuilder
+		self.players = players
 		gameOver = false
 		winner = false
 	}
 
 	public func playGame() {
 		initialScreenView()
 		while (!gameOver) {
-			playerTurn()
+			for player in players {
```

Comment:
We now have triple-nested logic here.  Can this be extracted into a method to make it a little easier to understand what's going on right away?